tests/test_trello.py

```diff
@@ -367,4 +368,39 @@ def test_actions_ordering(api_config, api_actions):  # pylint: disable=W0621
         previous_action_date = action_date
 
 
+def test_username_aliases(api_config, api_actions):  # pylint: disable=W0621
+    """Asserts Trello usernames are changed with configured aliases"""
+
+    unmodified_actions = copy.deepcopy(api_actions)
+
+    _ = api_actions
+
+    api = unit.TrelloAPI(
+        key=api_config['key'],
+        token=api_config['token'],
+        base_url=api_config['url'],
+    )
+
+    feed = unit.TrelloActivityFeed(
+        api=api,
+        board_id=api_config['board_id'],
+    )
+
+    unmuted_actions = [action for action in unmodified_actions if action['type'] in
+                       unit.TrelloActivityFeed.action_formatters.keys()]
+    unmuted_actions.reverse()
+
+    config_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'fixtures',
+                               'triscord.ini')
+
+    settings.CONFIG.read(config_path)
+    aliases = settings.CONFIG['Aliases']
+
+    for index, action in enumerate(feed.actions):
+        trello_username = unmuted_actions[index]['memberCreator']['username']
+        if trello_username in aliases:
+            assert action['memberCreator']['username'] != trello_username
```

Comment:
We probably want to support the case where the alias is identical to the trello username, to not put us an obstacle to generating the alias mapping automatically later down the line.